src/bp/ui-studio/src/web/views/OneFlow/diagram/nodes/SubworkflowContents/index.tsx

```diff
@@ -9,25 +9,28 @@ interface Props {
   node: BlockModel
   selectedNodeItem: () => { node: BlockModel; index: number }
   getCurrentLang: () => string
+  editNodeItem: (node: BlockModel, index: number) => void
 }
 
-const SubworkflowContents: FC<Props> = ({ node, selectedNodeItem, getCurrentLang }) => {
+const SubworkflowContents: FC<Props> = ({ node, selectedNodeItem, editNodeItem }) => {
   const { next } = node || {}
   const selectedContent = selectedNodeItem()
 
   return (
     <Fragment>
       <div className={style.contentsWrapper}>
         <div
+          onClick={() => editNodeItem(node, 0)}
```

Comment:
I know it's not part of the design but maybe we should mark  item as disabled if no input or no outputs vars defined. or add a visual cue. Preventing the user perform a useless click